frontend/components/songs/song_index.jsx

```diff
@@ -39,6 +40,16 @@ const SongIndex = ({
             setSongCardDropdownState({ isOpen: false });
         }
     }, [params]);
+    
+    if (currentViewRef && currentViewRef.current) {
+        if (songCardDropdownState.isOpen) {
+            // Prevent PlaylistShow from scrolling when dropdown is open
+                currentViewRef.current.style.overflowY = "hidden";
+            }
```

Comment:
Did you run prettier on this? Recommend keeping comments at the same indent level as the code, prefer this
```
if (...){
    // indented comment
   overflowY = "hidden"
}
```
over
```
if (...){
// indented comment
   overflowY = "hidden"
}
```

Reply:
Gotcha, thanks for the info. No, I think I forgot to run Prettier. Ty